lib/schemas/payment.js

```diff
@@ -0,0 +1,93 @@
+const {config} = require('bedrock');
+const {PaymentStatus} = require('bedrock-payment');
+
+const w3cDateTime = require('bedrock-validation/schemas/w3cDateTime');
```

Comment:
Don't access schemas through module loader, use the bedrock-validation schemas API. https://github.com/digitalbazaar/bedrock-validation/blob/master/lib/index.js#L21

Renaming a file internal to a module, in this case, bedrock-validation, should not be a breaking change if the public API remains unchanged.

See: https://github.com/digitalbazaar/bedrock-ledger-node/blob/master/schemas/webledger-validator.js#L7

Reply:
Addressed (hopefully correctly) here: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/3dbb7eb8ce3be7e2d9e6d23ef1efbb2c6a1b31a1